virtusize/src/main/java/com/virtusize/android/ui/VirtusizeWebViewFragment.kt

```diff
@@ -148,41 +149,50 @@ class VirtusizeWebViewFragment : DialogFragment() {
                     resultMsg: Message,
                 ): Boolean {
                     if (resultMsg.obj != null && resultMsg.obj is WebView.WebViewTransport) {
-                        val popupWebView = WebView(view.context)
-                        popupWebView.settings.javaScriptEnabled = true
-                        popupWebView.settings.javaScriptCanOpenWindowsAutomatically = true
-                        popupWebView.settings.setSupportMultipleWindows(true)
-                        popupWebView.settings.userAgentString = System.getProperty("http.agent")
-                        popupWebView.webViewClient =
-                            object : WebViewClient() {
-                                override fun shouldOverrideUrlLoading(
-                                    view: WebView,
-                                    url: String,
-                                ): Boolean {
-                                    if (VirtusizeURLCheck.isExternalLinkFromVirtusize(url)) {
-                                        val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
-                                        try {
-                                            startActivity(intent)
-                                        } finally {
-                                            return true
+                        val popupWebView =
+                            WebView(view.context).apply {
```

Comment:
I refactored it a bit using the scope function `apply`
For object configuration, we can use `apply`